mail/templates/applicant_letter/body.html

```diff
@@ -14,8 +15,8 @@
                   <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
                       <p style="margin: 0 0 10px;">
                         Sincerely,<br />
-                        <img src="{{ base_url }}/static/images/signature-mariah-rawding.png" alt="Signature for Mariah Rawding" /><br />
-                        Mariah Rawding<br />
+                        <img src="{{ base_url }}{% if signatory.image %}{% image_version_url signatory.image "fill-300x300" %}{% else %}{% static "images/signature-mariah-rawding.png" %}{% endif %}" alt="Signature for {% if signatory.name %}{{ signatory.name }}{% else %}Mariah Rawding{% endif %}" /><br />
```

Comment:
Can we change the `{% static "images/signature-mariah-rawding.png" %}` back to `/static/images/signature-mariah-rawding.png`? If cloudfront is enabled, the static tag will generate an absolute URL, so we need to make sure it's a relative URL so that it works with `base_url`.